### PR TITLE
Remove support for Facebook and Twitter sharing from Drop flow

### PR DESCRIPTION
**Pull request by Boerworz**
This commit also means that Allihoopa-iOS now works with (/requires) AllihoopaCore 4.0.0.

### CODE REVIEW

File: Allihoopa/Drop/DropViewController.m [hunk 1]
```diff
@@ -391,7 +391,7 @@ - (void)segueToProgressViewController {
 	[_infoViewController segueToProgressViewController];
 }
 
-- (void)segueToErrorViewController {
+- (void)segueToErrorViewController:(NSError *)error {
```
Comment (JagCesar) [inline Allihoopa/Drop/DropViewController.m]: Is this really related to this PR? Any why is this argument necessary?

Comment (JagCesar) [reply Allihoopa/Drop/DropViewController.m]: nvm 👯‍♂️

Comment (Boerworz) [reply Allihoopa/Drop/DropViewController.m]: The method signature of this method was changed to take the error argument in version 4.0.0 of `AllihoopaCore`, which we're using as of this pull request.
